allennlp/modules/augmented_lstm.py

```diff
@@ -64,16 +66,17 @@ def __init__(self,
             self.input_linearity = torch.nn.Linear(input_size, 4 * hidden_size, bias=True)
             self.state_linearity = torch.nn.Linear(hidden_size, 4 * hidden_size, bias=True)
 
-    def forward(self, inputs: PackedSequence,  # pylint: disable=arguments-differ
-                initial_state: Optional[Tuple[torch.Tensor, torch.Tensor]] = None):
+    def forward(self,  # pylint: disable=arguments-differ
+                inputs: PackedSequence,
+                hx: Optional[Tuple[torch.Tensor, torch.Tensor]] = None):
         """
         Parameters
         ----------
         inputs : PackedSequence, required.
             A tensor of shape (batch_size, num_timesteps, input_size)
             to apply the LSTM over.
 
-        initial_state : Tuple[torch.Tensor, torch.Tensor], optional, (default = None)
+        hx : Tuple[torch.Tensor, torch.Tensor], optional, (default = None)
```

Comment:
to match the pytorch ones as I thought I needed to pass keyword args somewhere originally. changed.